.github/workflows/build-and-test.yml

```diff
@@ -42,14 +42,14 @@ jobs:
         mkdir run
         echo "eula=true" > run/eula.txt
         timeout 90 ./gradlew runServer 2>&1 | tee -a server.log || true
-      if: INPUT_TIMEOUT == 0
+      if: env.INPUT_TIMEOUT == 0
```

Comment:
Rather than copy and pasting the same section twice, I'd much rather have an environment variable with a default timeout, or a separate step that sets it if it doesn't exist.

Reply:
I'm also pretty sure if an env variable doesn't exist it won't be ==0, there's a separate operator to check for that iirc

Reply:
ok looks like nulls are implicitly cast to a 0, nevermind <https://docs.github.com/en/actions/learn-github-actions/expressions>